package.json

```diff
@@ -8,38 +8,37 @@
     "lint:fix": "eslint --max-warnings 0 \"./**/*.{js,jsx,ts,tsx}\" --fix"
   },
   "dependencies": {
-    "@cosmjs/amino": "^0.29.5",
-    "@cosmjs/crypto": "^0.29.5",
-    "@cosmjs/encoding": "^0.29.5",
-    "@cosmjs/ledger-amino": "^0.29.5",
-    "@cosmjs/math": "^0.29.5",
-    "@cosmjs/proto-signing": "^0.29.5",
-    "@cosmjs/stargate": "^0.29.5",
-    "@cosmjs/utils": "^0.29.5",
-    "@keplr-wallet/types": "^0.9.0-alpha.4",
-    "@ledgerhq/hw-transport-webusb": "^6.24.1",
-    "axios": "^0.21.1",
-    "copy-to-clipboard": "^3.3.1",
-    "faunadb": "^4.1.1",
-    "next": "^12.1.6",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "react-select": "^5.2.2",
-    "uuid": "^8.3.0"
+    "@cosmjs/amino": "^0.30.1",
+    "@cosmjs/crypto": "^0.30.1",
+    "@cosmjs/encoding": "^0.30.1",
+    "@cosmjs/ledger-amino": "^0.30.1",
+    "@cosmjs/math": "^0.30.1",
+    "@cosmjs/proto-signing": "^0.30.1",
+    "@cosmjs/stargate": "^0.30.1",
+    "@cosmjs/utils": "^0.30.1",
+    "@keplr-wallet/types": "^0.11.56",
+    "@ledgerhq/hw-transport-webusb": "^6.27.13",
+    "axios": "^1.3.5",
+    "copy-to-clipboard": "^3.3.3",
+    "faunadb": "^4.8.0",
+    "next": "^13.3.0",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-select": "^5.7.2",
+    "uuid": "^9.0.0"
```

Comment:
Looks like this can just be removed (here or separately): https://github.com/cosmos/cosmos-multisig-ui/issues/92

Reply:
Yes, I noticed. `@keplr-wallet/types` is also unused for now but will probably be useful when improving keplr interaction.